spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -57,118 +57,233 @@ If you have debug mode turned on, you can get even more information, in a line s
 
 To build your own timestamp task, follow each of these procedures:
 
-. <<task-dev-guide-building-timestamp-task-setting-up-environment>>
-. <<task-dev-guide-building-timestamp-task-getting-started>>
-. <<task-dev-guide-building-timestamp-task-making-boot-task>>
+. <<task-dev-guide-building-timestamp-task-from-initializer>>
 . <<task-dev-guide-building-timestamp-task-recording-error>>
 . <<task-dev-guide-building-timestamp-task-adding-pre-post-processing>>
 . <<task-dev-guide-building-timestamp-task-bonus-adding-mysql>>
+. <<task-dev-guide-building-timestamp-adding-tasks-to-dataflow>>
 
-[[task-dev-guide-building-timestamp-task-setting-up-environment]]
-=== Setting up the Environment
+[[task-dev-guide-building-timestamp-task-from-initializer]]
+== Developing Your Timestamp Application
 
-Before we can do anything else, we have to set up our work environment. First, we must install RabbitMQ, as described https://github.com/cppwfs/DNDataflow/blob/master/labs/InstallRabbitnMQ.pdf[in this document].
-Then we need to create a directory and add a starter project (which we created to simplify this setup). To do so:
+So now that we've used a pre-built timestamp task application, let's create one of our own.
+Most IDEs have good support for Apache Maven, so we use it as the build tool for this project.
 
-. From the command line, run `git clone https://github.com/cppwfs/DNDataflow.git`.
-. From the command line, run `cd DNDataflow`.
+Before we begin, open a terminal to check that you have valid versions of Java and Maven
+installed, as shown in the following two listings:
 
-[[task-dev-guide-building-timestamp-task-getting-started]]
-=== Getting Started
+[source]
+$ java -version
+java version "1.8.0_31"
+Java(TM) SE Runtime Environment (build 1.8.0_31-b13)
+Java HotSpot(TM) 64-Bit Server VM (build 25.31-b07, mixed mode)
 
-To get started on making a task, we must first make a Spring Boot application. The DNDataflow project that we cloned in the preceding section took care of that for us, but we need to build and run it, as follows:
+[source]
+$ mvn -v
```

Comment:
You shouldn't need Maven installed since a project from Initializr comes with the wrapper...